lib/index.js

```diff
@@ -345,19 +345,19 @@ setup_brush = function(graph){
           .attr("transform", function(d, i) { return "translate(0,"+ i * 20 +")"; });
 
       legend.append("rect")
-          .attr("x", width + 20)
```

Comment:
But the width for the legend should be passed in ? I don't think we want to hardcode it.